newdle/client/src/components/summary/SummaryPage.js

```diff
@@ -65,6 +88,48 @@ export default function SummaryPage() {
               )}
             </Message>
           )}
+          {
+            <Modal
+              onClose={handleMailModalClose}
+              className={styles['user-search-modal']}
+              size="small"
+              closeIcon
+              open={mailModalOpen}
+            >
+              <Modal.Header className={styles['user-search-modal-header']}>
+                <span>E-mail participants </span>
+                <Label color="green" size="small" circular>
+                  {participantsWithEmail.length}
+                </Label>
+              </Modal.Header>
+              <Modal.Content>
+                {hasParticipantsWithoutEmail && (
+                  <>
+                    Some of your recipients do not have e-mail addresses and will not be contacted:
+                    <br />
+                    <List>
+                      {participantsWithoutEmail.map((p, index) => {
+                        return (
+                          <List.Item key={index}>
```

Comment:
It is discouraged to use the `index` as a key. Check if we have something better that uniquely identifies participants (`p.id` maybe)